Sprint-1/implement/max.js

```diff
@@ -1,4 +1,7 @@
 function findMax(elements) {
+  if (!Array.isArray(elements)) return -Infinity; // This handles non-array input
+  const numElements = elements.filter((item) => typeof item === "number"); // this checks if elements are numbers
+  return numElements.length > 0 ? Math.max(...numElements) : -Infinity; // math.max  returns the max number
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```